.github/actions/spell-check/expect.txt

```diff
@@ -521,6 +500,7 @@ FZE
 gacutil
 Gaeilge
 Gaidhlig
+GC'ed
```

Comment:
check-spelling's handling of short words has changed a bit because of an edge case (the base here is 2 letters which is below the general 3 letter threshold, but the word is 5 letters which is clearly above).